eva-accession-import-dbsnp2/src/test/java/uk/ac/ebi/eva/accession/dbsnp2/test/BatchTestConfiguration.java

```diff
@@ -44,6 +45,7 @@
         ImportDbsnpJsonVariantsStepConfiguration.class,
         ImportDbsnpJsonVariantsReaderConfiguration.class,
         JsonNodeToClusteredVariantProcessorConfiguration.class,
+        ImportDbsnpJsonVariantsWriterConfiguration.class,
```

Comment:
Remember to add tests for the writer, and for the whole job. You can use tests in `eva-accession-import` as a reference.